torch_glow/tests/nodes/quantize_dequantize_test.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_quantization_basic():
+    """Basic test of the PyTorch quantize and dequantize Node on Glow."""
+
+    # This only test if operator has been successfully projected
+    # Calculating process in glow will be optimized
+    def test_simple(a, b):
+        q = torch.nn.quantized.Quantize(1, 0, torch.quint8)
+        dq = torch.nn.quantized.DeQuantize()
+        return dq(q(a.add(b)))
```

Comment:
add a todo to move the add into the lowered graph?

Reply:
I have added some comments before this test_simple already, but well add more wont bite ;)